qt/aqt/tts.py

```diff
@@ -257,11 +257,9 @@ def _on_done(self, ret: Future, cb: OnDoneCallback) -> None:
         # inject file into the top of the audio queue
         from aqt.sound import av_player
 
+        av_player.current_player = None
         av_player.insert_file(self.tmppath)
 
-        # then tell player to advance, which will cause the file to be played
-        cb()
-
```

Comment:
Just a heads up that MacTTSFilePlayer is not the default player on macOS. When I manually adjust the bottom of sound.py to use this player instead, things work correctly with this change though. :-)